### ISSUE TITLE
bug(material-experimental/mdc-snackbar): JAWS does not read mdc-snackbar message in Firefox

### ISSUE THREAD

**Issue by toastking** [angular/components#21137]
#### Reproduction

https://stackblitz.com/edit/mdc-snackbar-bug?file=src/app/app.component.ts

Steps to reproduce:
1. Click on the button to open the snackbar with JAWS enabled
 
#### Expected Behavior

JAWS should read out the snackbar message of "Hello World" after the button is clicked.

#### Actual Behavior

JAWS does not read out anything.

#### Environment

- Angular: 10.2.3
- CDK/Material: material-experimental 11.0
- Browser(s): Firefox 81.0.2 (64 bit)
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10
